File: app/src/main/java/com/shentu/wallpaper/model/entity/MicroUser.java
package com.shentu.wallpaper.model.entity;

import com.google.gson.annotations.SerializedName;

public class MicroUser {
    @SerializedName("id")
    public Integer uid;
    public String nickname;
    public String avatar;
    public String phone;
    public String email;
    public String signature;
    public String date_joined;
    public String last_login;
    public Integer sex;//默认保密
    public Integer pea = 0;
    public String token = "";

    @Override
    public String toString() {
        return "uid:" + uid + "nickname:" + nickname + "avatar:" + avatar + "phone:" + phone + "email:" + email;
    }
}

File: app/src/main/java/com/shentu/wallpaper/model/entity/MicroUser.kt
package com.shentu.wallpaper.model.entity

data class MicroUser(
        var avatar: String = "",
        var email: String = "",
        var nPrice: Int = 1,
        var nickname: String = "",
        var oPrice: Int = 3,
        var pea: Int = 0,
        var phone: String = "",
        var sex: Int = 0,
        var signature: String = "",
        var token: String = "",
        var uid: Int = 0,
        var vip: Boolean = false,
        var showDonateInterval: Long = 24 * 60 * 60 * 1000//一天
)